### PR TITLE
Change queue names to allow Helix redirect

### PR DESCRIPTION
**Pull request by ulisesh**
Now that we have completed https://github.com/dotnet/core-eng/issues/11006 we need to change the names of the queues to allow Helix redirect arcade jobs

### CODE REVIEW

File: azure-pipelines-richnav.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ stages:
         timeoutInMinutes: 90
         pool:
           name: NetCorePublic-Pool
-          queue: BuildPool.Server.Amd64.VS2017.Arcade.Open
+          queue: BuildPool.Server.Amd64.VS2017.Open
```
Comment (MattGal) [inline azure-pipelines-richnav.yml:28]: This implies there should be a queue named BuildPool.Server.Amd64.VS2017.Open.Arcade to go to, right?

Comment (ulisesh) [reply azure-pipelines-richnav.yml:28]: Yes

Comment (ulisesh) [reply azure-pipelines-richnav.yml:28]: Wait, my change is wrong. We shouldn't redirect build jobs

Comment (MattGal) [reply azure-pipelines-richnav.yml:28]: ah k.  Making more sense.

**Review by MattGal — APPROVED** [review]

**Pull request merged**